manticore/native/memory.py

```diff
@@ -892,6 +892,28 @@ def perms(self, index):
         else:
             return self.map_containing(index).perms
 
+    def max_access_size(self, addr_start, max_size, access):
+        """
+        Finds maximum executable memory size
+        starting from addr and up to max_size.
+
+        :param addr_start: the starting address.
+        :param size: the maximum size.
+        :param access: the wished access.
+        """
+        size = 0
+        addr = addr_start
```

Comment:
The `addr_start` is not used later on, so we could have it as the argument name, reuse it and assign the ending address instead:

```python
        max_addr = addr + max_size
        while addr < max_addr:
```

...my reasoning is that it is Python and the `addr_start + max_size` won't be optimized out (at least not in or by CPython VM):

![image](https://user-images.githubusercontent.com/10009354/64336017-53f79500-cfdc-11e9-9319-cf64a8cb14e8.png)

Reply:
ok